python-threatexchange/threatexchange/cli/tests/match_cmd_test.py

```diff
@@ -31,3 +34,33 @@ def test_invalid_hash(self):
             ("-H", "video", "--", not_hash),
             f"{not_hash!r} from .* is not a valid hash for video_md5",
         )
+
+    def test_non_photo_match_with_rotations(self):
+        with tempfile.NamedTemporaryFile() as f:
+            for content_type in ["url", "text", "video"]:
+                self.assert_cli_usage_error(
+                    ("--rotations", content_type, f.name),
+                    msg_regex="--rotations flag is only available for Photo content type",
+                )
+
+    def test_photo_hash_with_rotations(self):
+        test_file = pathlib.Path("threatexchange/tests/hashing/resources/rgb.jpeg")
+
+        hash_cmd = ThreatExchangeCLIE2eHelper()
+        hash_cmd.COMMON_CALL_ARGS = ("hash",)
+        hash_cmd._state_dir = pathlib.Path()
+
+        hash = hash_cmd.cli_call("photo", str(test_file))
+        assert hash == "pdq fb4eed46cb8a6c78819ca06b756c541f7b07ef6d02c82fccd00f862166272cda\n"
+
+        # rotated_images = PhotoContent.all_simple_rotations(test_file.read_bytes())
+
+        # img = rotated_images[RotationType.ROTATE90] #try with 1 rotated image first
+
+        # with tempfile.NamedTemporaryFile() as tmp_file:
+        #     img = rotated_images[RotationType.ROTATE90]
+        #     tmp_file.write(img)
+        #     self.assert_cli_output(
+        #         ("--rotations", "photo", "--", tmp_file.name),
+        #         "video_md5 - (Sample Signals) INVESTIGATION_SEED",
+        #     )
```

Comment:
My current approach for this test: 
1. Hash an image 
2. Fetch 
3. Call `match --rotations` the rotated version of the original image 

Q: @Dcallies 
* I got the hash pdq of the image (line 53) but how do i fetch indexes and call match command on this hash?
* If there is a match found, what should the output look like? When I try it with the CLI it keeps output 
`pdq 16 (Sample Signals) INVESTIGATION_SEED`

Reply:
If there is a match, then the output is the matching collaboration and distance.

Consider this ouptput:
> pdq 16 (Sample Signals) INVESTIGATION_SEED

| signal_type | distance | collab name | confidence |
|---| --- | --- | --- |
| pdq | 16 | Sample Signals | INVESTIGATION_SEED

Reply:
Thank you for your comment! I found the bug. I shouldn't have checked s_type == PhotoContent since it's SignalType not ContentType